pkg/plugins/plugins.go

```diff
@@ -199,24 +201,42 @@ func GetAll(imp string) ([]*Plugin, error) {
 		err error
 	}
 
-	chPl := make(chan plLoad, len(pluginNames))
+	driverExist := func(dn string) bool {
+		_, ok := ext[dn]
+		return ok
+	}
+
+	chPl := make(chan plLoad, 0)
```

Comment:
If doing a 0 buffer, no need to specify the 0... but I think we want to use a buffered channel here so we can wg.Wait later.